src/staticize.jl

```diff
@@ -52,3 +52,13 @@ dynamicize(A::PermMatrix) = PermMatrix(Vector(A.perm), Vector(A.vals))
 function dynamicize(A::SSparseMatrixCSC)
     SparseMatrixCSC(A.m, A.n, Vector(A.colptr), Vector(A.rowval), Vector(A.nzval))
 end
+
+function findnz(M::SDMatrix)
```

Comment:
why not overload the one in `SparseArrays`?

Reply:
it can not be defined for Diagonal matrix due to pirating.